src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/caseload/ApproverCaseloadService.kt

```diff
@@ -74,7 +87,7 @@ class ApproverCaseloadService(
         isDueForEarlyRelease = it.licenceSummary?.isDueForEarlyRelease,
         probationPractitioner = findProbationPractitioner(it.deliusRecord, it.comUsernameOnLicence, coms),
       )
-    }.sortedWith(compareBy(nullsLast()) { it.releaseDate })
+    }.sortedWith(compareBy(nullsFirst()) { it.releaseDate })
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** ‏Tests in the frontend indicate that any release date "not found" (in our case we send them through as nulls for the front end to handle as "not found) is surfaced above release dates when sorted so just changing the logic here.